src/Runtime/Distributions/Automata/ListAutomaton.cs

```diff
@@ -2,6 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Runtime.Serialization;
```

Comment:
I assume these are automatically inserted, but you probably want them inside the namespace { } definition for consistency.

Reply:
Sorry, I'm used to IDE doing the right thing (due to ReSharper settings at the root of the project) and never checked for this explicitly. Will fix.